live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -323,3 +323,33 @@
   with_items:
     - { key: 'kernel.core_pattern', value: '/var/crash/core.%e.%p.%t' }
     - { key: 'vm.overcommit_memory', value: '1' }
+
+#
+# Add the apt repos that have the debug versions of packages, so that we
+# can download a debug kernel to use with crash.
+#
+- apt_repository:
+    repo: "{{ item }}"
+  with_items:
+    - deb http://ddebs.ubuntu.com bionic main restricted universe multiverse
+    - deb http://ddebs.ubuntu.com bionic-updates main restricted universe multiverse
+    - deb http://ddebs.ubuntu.com bionic-proposed main restricted universe multiverse
+
+- shell: ls /boot/vmlinuz-* | sed 's|/boot/vmlinuz-||'
+  register: kernel_version
+
+- apt:
```

Comment:
I think it's likely that this task will be flaky. My experience with `ddebs.ubuntu.com` has been that it's slow and flaky as hell. It may even be faster to fetch from our Artifactory instance. In the short term I'd suggest doing a 3x retry here, in the long term I'd suggest we factor this into our existing plans to mirror the regular Ubuntu repositories.

Reply:
Great point. I've added the retries, and I'll file a task to tracking mirroring `ddebs.ubuntu.com`.

Reply:
> in the long term I'd suggest we factor this into our existing plans to mirror the regular Ubuntu repositories.

I think we'll need/want to do this for all the repositories we pull from; i.e. postgres, ansible, regular ubuntu, and now ddebs.ubuntu.